app/models/article.rb

```diff
@@ -0,0 +1,5 @@
+class Article < ApplicationRecord
+  belongs_to :user
+  validates :title, :body, :user_id, presence: true
+  validates :user_id, numericality: { only_integer: true }
```

Comment:
This is not needed in this case, rails will automatically provide validation to verify an existing user